pydantic_redis/abstract.py

```diff
@@ -21,6 +21,9 @@ class Config:
         arbitrary_types_allowed = True
         orm_mode = True
 
+def pydantic_serializer(data):
```

Comment:
We could move this function to the bottom of the module, below `class _AbstractModel` for a better aesthetic feel.